frontend-react/src/components/Admin/AdminFormEdit.tsx

```diff
@@ -40,13 +40,21 @@ export const TextAreaComponent = (params: {
     label: string;
     defaultvalue: object;
     savefunc: (val: object) => void;
+    defaultnullvalue: string | null;
 }): JSX.Element => {
     let defaultValue = JSON.stringify(params?.defaultvalue, undefined, 2);
-    if (defaultValue === "null" || defaultValue === "[]") {
+    if (
+        defaultValue === "null" ||
+        defaultValue === "[]" ||
+        defaultValue === "{}"
+    ) {
         defaultValue = "";
     }
 
     const key = params.fieldname;
+    const defaultnullvalue = params.defaultnullvalue
+        ? params.defaultnullvalue
+        : null;
```

Comment:
I thought if you pass `null` (versus `"null"`), then the call below to `JSON.parse()` could throw an exception... but I tested it and `JSON.parse(null)` doesn't fail, but `JSON.parse()` does. Odd.